src/subPages/Curation/index.js

```diff
@@ -0,0 +1,153 @@
+
+import { foundationPartial } from 'styles/foundation';
+
+import ipro from 'styles/interpro-new.css';
+import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+
+const f = foundationPartial(ipro, ebiGlobalStyles, fonts);
+
+/*:: type Props = {
+  data: Object,
+}; */
+
+const CurationSubPage = ({ data }) => {
+  const { loading, payload } = data;
+
+  if (loading) return <Loading />;
+  // eslint-disable-next-line camelcase
+  const details = payload?.extra_fields.details;
+
+  return (
+    <div className={f('row', 'column')}>
+      <h4>Curation</h4>
+      <table className={f('light', 'table-sum')}>
+        <tbody>
+          <tr>
+            <td>Author</td>
+            <td className={f('first-letter-cap')}>
+              {details.curation.authors
+                .map((author) => {
+                  let preferredName = author.author;
+                  if (author.orcid) {
+                    preferredName = (
+                      <a
+                        href={`https://orcid.org/${author.orcid}`}
+                        target="_blank"
+                        rel="noreferrer"
+                      >
+                        {author.author}
+                        <i
+                          className={f('icon', 'icon-common')}
+                          data-icon="&#xf112;"
+                        />{' '}
+                      </a>
+                    );
+                  }
+                  return <span key={author.author}>{preferredName}</span>;
+                })
+                .reduce((prev, curr) => [prev, ', ', curr])}
+            </td>
+          </tr>
+          <tr>
+            <td>Sequence Ontology</td>
+            <td className={f('first-letter-cap')}>
+              <a
+                href={`http://www.sequenceontology.org/miso/current_svn/term/${details.curation.sequence_ontology}`}
+                target="_blank"
+                rel="noreferrer"
+              >
+                {details.curation.sequence_ontology}
+              </a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h4>HMM Information</h4>
+      <table className={f('light', 'table-sum')}>
+        <tbody>
+          <tr>
+            <td>HMM build commands</td>
+            <td>
+              Build method: {details.hmm.commands.build}
+              <br />
+              Search method: {details.hmm.commands.search}
+            </td>
+          </tr>
+          <tr>
+            <td>Gathering threshold</td>
+            <td>
+              Sequence: {details.hmm.cutoffs.gathering.sequence}
+              <br />
+              Domain: {details.hmm.cutoffs.gathering.domain}
+            </td>
+          </tr>
+          <tr>
+            <td>Download</td>
+            <td>
+              <Link
+                href={`${config.root.API.href}/entry/pfam/${payload?.metadata?.accession}?annotation=hmm`}
```

Comment:
I changed the link to get the hmm from our API to reduce the use of pfam website